packages/driver/src/cypress/runner.ts

```diff
@@ -1803,6 +1860,10 @@ export default {
 
         return logs.push(attrs)
       },
+
+      getTotalTestsBeforeFilter () {
```

Comment:
a comment would be helpful here.

Reply:
Removed in [5e431a0](https://github.com/cypress-io/cypress/pull/25265/commits/5e431a05532e2116e048ad3468ede9aa41ff6eda)!